docs/streams.md

```diff
@@ -137,9 +136,7 @@ read a hard deleted stream, the read returns a `StreamDeleted` or `410` result.
 The events in the deleted stream are liable to be removed in a scavenge, but the tombstone event remains.
 
 A hard delete of a stream is permanent. You cannot append to the stream or recreate it. As such, you should
-generally soft delete streams unless you have a specific need to permanently delete the stream. On rare
-occasions when you need to re-open a hard-deleted stream, you can force EventStoreDB to scavenge it by using
-the [ignore hard deletes](operations.md#ignore-hard-delete) option.
```

Comment:
@timothycoleman that options is not there on 20.10 ?

Reply:
it is there is in 20.10, it's there in 22.10 as well and should behave in the same way, but it requires some care to use and could be a source of dataloss if the customer isn't careful. i've removed it from the doc to avoid people getting into difficulty with it